docs/website/data/integrations.yaml

```diff
@@ -690,6 +690,20 @@ integrations:
         link: https://www.youtube.com/watch?v=jrrW855xL3s
 
 
+  sysdig-image-scanner:
+    title: Kubernetes Sysdig Image Scanner admission controller
```

Comment:
nit: Can you make this "title case", eg `Kubernetes Sysdig Image Scanner Admission Controller`

Reply:
Sure, done!